assets/components/organisms/question/question.twig

```diff
@@ -0,0 +1,22 @@
+<div class="question">
+
+  <div class="question-img">
+    {% include 'atoms::picture/picture-question.twig' %}
+  </div>
+
+  <div class="question-content">
+    <p class="h3">À quoi l'éternel brouillard de San Francisco est-il dû ?</p>
+
+    <div class="question-answers">
+      {% for i in [1,2,3] %}
+          <input type="radio" id="custom-radio{{i}}" name="customRadio" class="custom-control-input">
+          <label class="custom-control-label" for="custom-radio{{i}}">
+            <span class="custom-control-label-content">
+            {{ label ?: 'Option one' }}</span>
```

Comment:
No needs of dynamic single label here if we don't have dynamic title. In this case, it will not suppose to be included in a dynamic way in some example page, so feel free to use something like `Option {{i}}` 😉